lib/experimental/class-theme-json-webfonts-handler.php

```diff
@@ -0,0 +1,537 @@
+<?php
+/**
+ * Handler for theme.json webfonts.
+ *
+ * @package    WordPress
+ * @subpackage WebFonts
+ * @since      6.0.0
+ */
+
+if ( ! function_exists( '_wp_theme_json_webfonts_handler' ) ) {
+	add_action( 'plugins_loaded', '_wp_theme_json_webfonts_handler' );
+	/**
+	 * Runs the theme.json webfonts handler to expose defined webfonts
+	 * to the editor's typography pickers and generate `@font-face` style
+	 * declarations.
+	 *
+	 * This is not a public API, but rather an internal handler.
+	 * A future public Webfonts API will replace this code.
+	 *
+	 * This code design is intentional to encapsulate the inner-workings
+	 * of this handler to ensure it is not accessible from outside of this
+	 * function. Why? This is a stopgap implementation to provide the mechanisms
+	 * for webfonts to be defined in a theme's `theme.json` file for style
+	 * variations.
+	 *
+	 * @link https://github.com/WordPress/gutenberg/issues/40472
+	 *
+	 * @since 6.0.0
+	 */
+	function _wp_theme_json_webfonts_handler() {
+		static $handler = null;
+
+		// This check prevents the class from being loaded into memory more than once.
+		if ( null !== $handler ) {
+			return;
+		}
+
+		/**
+		 * Handler for `theme.json` webfonts.
```

Comment:
```suggestion
		 * Handler for theme.json webfonts.
```

- Officially, markdown syntax is only valid the body of a docblock, not the title.
- Other docblock titles in the PR do not encapsulate theme.json at all.